### ISSUE TITLE
Amazons border markers doesn't differ much from player to player

### ISSUE THREAD

**Issue by frankystone** [widelands/widelands#5713]
The border markers for Amazons are visually close to equal with regard to player color. Here are three Amazons players, Yellow, Green and Red:

![shot0005](https://user-images.githubusercontent.com/6730556/208293094-6e1062bf-6b3c-4f71-9242-53f2ed6e9dc9.png)

One can't see which border is from which player (color). This can lead into situations where you don't know if the player in the east is the same than the player on the west.

**Describe the solution you'd like**
The player color should be more visible on Amazons border markers to make them better distinguishable.

**Describe alternatives you've considered**
Don't know :-)

**Additional context**
Maybe the Amazons "flags" should also be reworked to make the player color more prominent.

**Implementation steps**
- [ ] 
- [ ] 
- [ ]

**Comment by tothxa** [comment]
+1, for the flags too. I think at least the carriers' dress should have some part in player colour too.

**Comment by Noordfrees** [comment]
How about this for the frontiers:
![grafik](https://user-images.githubusercontent.com/48293446/208295761-c6aea422-2bc8-411c-854d-e7ddd357671e.png)
![grafik](https://user-images.githubusercontent.com/48293446/208295785-22459808-bb2c-4466-a292-bed802b12e29.png)
